dlt/normalize/normalize.py

```diff
@@ -67,13 +69,13 @@ def load_or_create_schema(schema_storage: SchemaStorage, schema_name: str) -> Sc
 
     @staticmethod
     def w_normalize_files(
-            normalize_storage_config: NormalizeStorageConfiguration,
-            loader_storage_config: LoadStorageConfiguration,
-            destination_caps: DestinationCapabilitiesContext,
-            stored_schema: TStoredSchema,
-            load_id: str,
-            extracted_items_files: Sequence[str],
-        ) -> TWorkerRV:
+        normalize_storage_config: NormalizeStorageConfiguration,
+        loader_storage_config: LoadStorageConfiguration,
+        destination_caps: DestinationCapabilitiesContext,
+        stored_schema: TStoredSchema,
+        load_id: str,
+        extracted_items_files: Sequence[str],
+    ) -> TWorkerRV:
```

Comment:
you must create load storage with a correct file format later
```
load_storage = LoadStorage(False, destination_caps.preferred_loader_file_format, LoadStorage.ALL_SUPPORTED_FILE_FORMATS, loader_storage_config)
```
goes for puae-json files

the parquet must have a parquet extension as preferred.

destination_caps have information if parquet is supported at all. if not you must raise a really descriptive exception why we can't do parquet for given destination.

we could OFC convert parequet to py_list and stream it into the desired writer like we stream jsonl files. but I think this is not the reason to use pyarrow directy. IMO a nice exception is enough.

this case also needs to be tested